connector-packager/tests/test_resources/mcd_field_not_in_normalizer/connectionMetadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<connection-metadata>
+  <database enabled='true' label='Database'>
```

Comment:
I think we do this as a follow-up item, but in many cases database enabled means 'dbname' should also be included as a required attribute.  We'll need to review the use cases I think.

Reply:
Opened TFS 1300614 as follow up